lib/internal/test_runner/harness.js

```diff
@@ -9,34 +13,43 @@ const {
     ERR_TEST_FAILURE,
   },
 } = require('internal/errors');
-const { Test } = require('internal/test_runner/test');
+const { Test, ItTest, Suite } = require('internal/test_runner/test');
+
 
-function createProcessEventHandler(eventName, rootTest, testResources) {
+const testResources = new SafeMap();
+const root = new Test({ name: '<root>' });
```

Comment:
We don't want the root test options to depend on the mutable `Object.prototype`.
```suggestion
const root = new Test({ __proto__: null, name: '<root>' });
```